lightkurve/tests/test_periodogram.py

```diff
@@ -201,7 +204,8 @@ def test_flatten():
     assert all(b.power == p.smooth(method='logmedian', filter_width=0.01).power)
     assert all(s.power == p.flatten().power)
     str(s)
-    s.plot()
+    ax = s.plot()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'ax' is assigned to but never used](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4938792)